src/crosshook-native/src/styles/library.css

```
@@ -517,3 +563,148 @@
+
+.crosshook-game-inspector__profile-summary {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  font-size: 0.8rem;
+  color: var(--crosshook-color-text);
+}
+
+.crosshook-game-inspector__profile-line {
+  margin: 0;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)